overlord/snapstate/snapstate.go

```diff
@@ -2272,11 +2343,14 @@ func UpdateWithDeviceContext(st *state.State, name string, opts *RevisionOptions
 		return opts, flags, &snapst
 	}
 
-	_, tts, err := doUpdate(context.TODO(), st, []string{name}, toUpdate, params, userID, &flags, deviceCtx, fromChange)
+	_, updateTss, err := doUpdate(context.TODO(), st, []string{name}, toUpdate, params, userID, &flags, deviceCtx, fromChange)
 	if err != nil {
 		return nil, err
 	}
 
+	// pre-download tasksets are only used for auto-refreshes
```

Comment:
Maybe expand the comment slightly: `// pre-download tasksets are only used for auto-refreshes so they are ignored here`  (unless I misunstand the code?

Reply:
Done, thank you for the review